products/db-checkbook/sql/parks.sql

```diff
@@ -0,0 +1,40 @@
+-- assign geometries to capital projects using Parks Properties dataset
+
+WITH master AS (
+    SELECT 
+        a.fms_id,
+        a.agency, 
+        a.budget_code, 
+        a.contract_purpose, 
+        a.has_geometry, 
+        a.geometry, 
+        b.EAPPLY, 
+        b.ADDRESS, 
+        b.NAME311, 
+        b.SIGNNAME, 
+        b.multipolygon
+FROM csdb a
+JOIN parks b ON (
+        (upper(a.agency) LIKE '%DEPARTMENT OF PARKS AND RECREATION%' OR
+        upper(a.agency) LIKE '%DPR%')
+        AND a.geometry IS NULL
+        AND (
+            upper(a.budget_code) LIKE '%' || upper(b.EAPPLY) || '%' OR
+            upper(a.budget_code) LIKE '%' || upper(b.ADDRESS) || '%' OR
+            upper(a.budget_code) LIKE '%' || upper(b.NAME311) || '%' OR
+            upper(a.budget_code) LIKE '%' || upper(b.SIGNNAME) || '%' OR
+            upper(a.contract_purpose) LIKE '%' || upper(b.EAPPLY) || '%' OR 
+            upper(a.contract_purpose) LIKE '%' || upper(b.ADDRESS) || '%' OR
+            upper(a.contract_purpose) LIKE '%' || upper(b.NAME311) || '%' OR
```

Comment:
I'm not sure we need these other fields - from what I can tell, EAPPLY and NAME311 seem like duplicates of SIGNNAME (other than a case I saw where EAPPLY is blank). We also definitely need the 
```sql
WHERE
      upper(b.signname) <> 'PARK' AND
      upper(b.signname) <> 'LOT' AND
      upper(b.signname) <> 'GARDEN' AND
      upper(b.signname) <> 'TRIANGLE' AND
      upper(b.signname) <> 'SITTING AREA'
```

from looking at the data. Multiple "Park"s and "Garden"s which would be matching on to tons of projects that they shouldn't

<img width="329" alt="image" src="https://github.com/NYCPlanning/data-engineering/assets/9454672/c4b9846a-f239-448b-93b1-7a1175c84ecd">